components/script/dom/document.rs

```diff
@@ -1587,9 +1614,14 @@ impl DocumentMethods for Document {
         self.encoding_name.borrow().clone()
     }
 
+    //https://dom.spec.whatwg.org/#dom-document-charset
```

Comment:
Put a space after the double-slash so that it looks like the following:
```
    // https://dom.spec.whatwg.org/#dom-document-charset
```
That should fix the tidy issue.